### DISCUSSION TITLE
Target-typed new expressions

### DISCUSSION THREAD

**Discussion by MartyIX** [JosefPihrt/Roslynator#785] [Ideas]
Hi,

can Roslynator help me convert my codebase to use the new [target-typed new expressions](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/target-typed-new)?

Thank you!

**Comment by josefpihrt** [comment]
Sould like a good idea!

https://github.com/JosefPihrt/Roslynator/issues/786
